lib/prm/web/views/error_helpers.ex

```diff
@@ -0,0 +1,22 @@
+defmodule PRM.Web.ErrorHelpers do
+  def translate_error({msg, opts}) do
```

Comment:
The module was brought back because it is required by `PRM.Web.ChangesetView`, which in turn is required by `lib/prm/web/controllers/fallback_controller.ex` that renders errors.

I brought them momentarily because the tests were not passing, and I was just following happy path from Phoenix.

I think they can ultimately be removed since we're using EView. I'd love some guidance here.